rhobs/alerting/data_plane/prometheus.availability_metric_konflux_alerts.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-konflux-up-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: konflux_up_alerts
+    interval: 1m
+    rules:
+    - alert: KonfluxAlert
+      expr: (konflux_up offset 1h) unless (konflux_up)
+      for: 1m
+      labels:
+        severity: warning
+      annotations:
+        summary: Availability metric 'konflux_up' is missing.
+        description: >-
+          Availability metric 'konflux_up' is missing some entries for check
+          {{ $labels.check }} on service {{ $labels.service }} when compared to
+          previous hour.
```

Comment:
I think we should also include the `source_cluster` label in here somewhere as we have done with other alerting rules.